Spanner/src/Database.php

```diff
@@ -1784,7 +1787,7 @@ public function executePartitionedUpdate($statement, array $options = [])
         try {
             return $this->operation->executeUpdate($session, $transaction, $statement, [
                 'statsItem' => 'rowCountLowerBound'
-            ] + $options);
+            ] + $options, $transaction->getIdReference());
```

Comment:
Should the transaction id reference be a part of options array?

Reply:
(might not be possible if you are relying on pass by reference to save transaction id)

Reply:
I thought about this and sending reference in an array works. But this parameter not really comes under the meaning of `options` is what I felt. transaction Id is fine, but a reference to a transactionId, not sure! If you feel like its okay to pass in the array, I can change in all the functions to pass this by reference in the options array. Please let me know what do you think?

Reply:
I don't think passing variable by reference in options array is good.

Reply:
We could use `transaction` key in `$option` like this:

 `$options['transaction'] = ['id' => $transactionId];`
 OR
`$options['transaction'] = ['id' => 'begin'];`

Reply:
We need to update the transactionId of the Transaction object from the Result class obnect, thats were reference of this attribute or the object is required!

Reply:
This logic is changed now, hence resolving the comment.